panel/_templates/nb_template.html

```diff
@@ -0,0 +1,43 @@
+{#
+Renders Bokeh models into a basic .html file.
+
+:param title: value for ``<title>`` tags
+:type title: str
+
+:param plot_resources: typically the output of RESOURCES
+:type plot_resources: str
+
+:param plot_script: typically the output of PLOT_SCRIPT
+:type plot_script: str
+
+Users can customize the file output by providing their own Jinja2 template
+that accepts these same parameters.
+
+#}
+
+{% from macros import embed %}
+
+{% block inner_head %}
+  {% block preamble %}{% endblock %}
+  {% block resources %}
+    {% block js_resources %}
+      {{ bokeh_js | indent(8) if bokeh_js }}
+    {% endblock %}
+  {% endblock %}
+{% endblock %}
+{% block postamble %}{% endblock %}
+{% block body %}
+  {% block inner_body %}
+    {% block contents %}
+      {% for doc in docs %}
+        {{ embed(doc) if doc.elementid }}
+        {% for root in doc.roots %}
+          {% block root scoped %}
+            {{ embed(root) | indent(10) }}
+          {% endblock %}
+        {% endfor %}
+      {% endfor %}
+    {% endblock %}
+    {{ plot_script | indent(8) }}
+  {% endblock %}
+{% endblock %}
```

Comment:
Is there a reason for copying this, instead of loading this template from bokeh? Then it wouldn't be necessary it reimplement `html_page_for_render_items()`.

Reply:
Yes, this template is stripped from all the standalone HTML page boilerplate including `<html>`, `<head>`, `<body>` tags.